src/main/app/claims/ccjModelConverter.ts

```diff
@@ -32,9 +50,73 @@ function convertPayBySetDate (draftCcj: DraftCCJ): Moment {
     ? draftCcj.payBySetDate.date.toMoment() : undefined
 }
 
+function getDateOfBirth (defendant: Party): Moment {
+  if (defendant.type === PartyType.INDIVIDUAL.value) {
+    return MomentFactory.parse((defendant as Individual).dateOfBirth)
+  }
+  return undefined
+}
+
+function getDefendantPaymentIntention (claim: Claim): PaymentIntention {
+  const response: FullAdmissionResponse | PartialAdmissionResponse = claim.response as FullAdmissionResponse | PartialAdmissionResponse
+  const paymentIntention = new PaymentIntention()
+
+  if (response.paymentIntention) {
+    paymentIntention.paymentOption = new PaymentOption(PaymentType.valueOf(response.paymentIntention.paymentOption))
+    paymentIntention.paymentDate = response.paymentIntention.paymentDate && new PaymentDate(
+      new LocalDate(response.paymentIntention.paymentDate.year(), response.paymentIntention.paymentDate.month(), response.paymentIntention.paymentDate.date())
+    )
+
+    const repaymentPlan: ResponseRepaymentPlan = response.paymentIntention.repaymentPlan
+
+    paymentIntention.paymentPlan = repaymentPlan && new PaymentPlan(claim.totalAmountTillToday,
+      repaymentPlan.instalmentAmount,
+      new LocalDate(repaymentPlan.firstPaymentDate.year(), repaymentPlan.firstPaymentDate.month(), repaymentPlan.firstPaymentDate.date()),
+      PaymentSchedule.of(repaymentPlan.paymentSchedule)
+    )
+  }
+  return paymentIntention
+}
+
+function convertAlreadyPaidAmount (claim: Claim, claimantResponse: DraftClaimantResponse): number {
+  switch (claim.response.responseType) {
+    case ResponseType.PART_ADMISSION:
+      return claimantResponse.paidAmount.amount
+    case ResponseType.FULL_ADMISSION:
+      return claim.claimData.amount.totalAmount()
+    default:
+      throw new Error(`Incompatible response type: ${claim.response.responseType}`)
+  }
+}
+
 export class CCJModelConverter {
 
-  static convert (draft: DraftCCJ): CountyCourtJudgment {
+  static convertForIssue (claim: Claim, draft: Draft<DraftClaimantResponse>): CountyCourtJudgment {
+    const claimantResponse: DraftClaimantResponse = draft.document
+
+    const defendantPaymentMethodAccepted = claimantResponse.acceptPaymentMethod.accept.option === YesNoOption.YES.option
+
+    const paymentIntention: PaymentIntention = defendantPaymentMethodAccepted ? getDefendantPaymentIntention(claim)
+      : claimantResponse.alternatePaymentMethod
+
+    let repaymentPlan: RepaymentPlan
+    let payBySetDate: Moment
+    let paymentOption: string
+
+    if (paymentIntention) {
```

Comment:
Please remove, payment intention must always be there. Only question is around source of data.